src/main/resources/com/coravy/hudson/plugins/github/GithubProjectProperty/help-statusContext.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
+        This value will be used as context name for
+        <a href="https://developer.github.com/v3/repos/statuses/">commit status</a> if status builder or 
+        status publisher is defined for this project.
+    </p>
+
+    <p>
+        If you leave it empty, job name will be used for builder and publisher.
+    </p>
+</div>
```

Comment:
AFAIK keeping NL is an approach from history - you should have end marker for streams of data. Though today it not critical for high level langs :)